Formula/gpsd.rb

```diff
@@ -1,9 +1,10 @@
 class Gpsd < Formula
   desc "Global Positioning System (GPS) daemon"
   homepage "http://catb.org/gpsd/"
```

Comment:
Can either update here or separate syntax PR, but `homepage` redirects (after 5 seconds) to new page https://gpsd.gitlab.io/gpsd/index.html